### ISSUE TITLE
Features broken on new PR merge box beta 

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#8123]
- https://github.com/orgs/community/discussions/143787
- https://github.blog/changelog/2024-12-03-improved-pull-request-merge-experience-now-in-public-preview/

One thing to note is that `clean-checks-list` is now implemented natively 🎉 

The first step here is to figure out what's broken exactly.

**Comment by kovsu** [comment]
- [x] `clear-pr-merge-commit-message`
	- new bug opened https://github.com/refined-github/refined-github/issues/8246
- [x] `closing-remarks` (specifically after clicking the merge button)
- [x] `cross-deleted-pr-branches`
- [x] `pr-branch-auto-delete`
- [x] `update-pr-from-base-branch`
- [x] `pr-base-commit`
- [x] https://github.com/refined-github/refined-github/issues/8902
- [x] `suggest-commit-title-limit`
- [ ] ~~`convert-pr-to-draft-improvements`~~ 👋

**Comment by fregante** [comment]
I'll also list features that are no longer needed:

- `clean-checks-list`
- `prevent-pr-merge-panel-opening`

**Comment by jpike88** [comment]
pr-branch-auto-delete is the most pressing one, can that be prioritized

**Comment by fregante** [comment]
@jpike88 done.

`update-pr-from-base-branch` would be another one that's super important.

We should just add a whole section to the box instead of trying to extend the existing message _as natively as possible,_ because that solution was unmaintainable:

- https://github.com/refined-github/refined-github/issues/6574

**Comment by fregante** [comment]
@alanconway this repo is for a browser extension. You should report that to GitHub

**Comment by vintagesucks** [comment]
After #8193, I see this in a private repo. Should I open a new issue?

![Image](https://github.com/user-attachments/assets/c30518f9-3db4-458a-8160-4cbffb745385)

Without Refined GitHub, it looks like this:

![Image](https://github.com/user-attachments/assets/e3739644-493d-4c00-bf3f-94e1538ed604)

---

<details>
<summary>DOM without `update-pr-from-base-branch` (25.1.24)</summary>

```html
<section aria-label="Conflicts" class="border-bottom borderColor-muted" aria-describedby=":r10:"><div class="d-flex flex-column width-full overflow-hidden"><div class="MergeBoxSectionHeader-module__wrapper--f99Ts flex-column flex-sm-row flex-items-center flex-sm-items-start flex-justify-between"><div class="d-flex width-full"><div class="mr-2 flex-shrink-0"><div overflow="hidden" size="32" class="Box-sc-g0xbh4-0 iAmUFw"><div display="flex" size="32" class="Box-sc-g0xbh4-0 jneZjk"><svg aria-hidden="true" focusable="false" class="octicon octicon-alert-fill" viewBox="0 0 16 16" width="16" height="16" fill="currentColor" style="display: inline-block; user-select: none; vertical-align: text-bottom; overflow: visible;"><path d="M6.457 1.047c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0 1 14.082 15H1.918a1.75 1.75 0 0 1-1.543-2.575ZM8 5a.75.75 0 0 0-.75.75v2.5a.75.75 0 0 0 1.5 0v-2.5A.75.75 0 0 0 8 5Zm1 6a1 1 0 1 0-2 0 1 1 0 0 0 2 0Z"></path></svg></div></div></div><div class="d-flex flex-1 flex-column flex-sm-row gap-2"><div class="flex-1"><h3 class="Box-sc-g0xbh4-0 isSOdJ prc-Heading-Heading-6CmGO" id=":r10:">This branch is out-of-date with the base branch</h3><p class="fgColor-muted mb-0 ">Merge the latest changes from develop into this branch. This merge commit will be associated with vintagesucks.</p></div><div class="Box-sc-g0xbh4-0 prc-ButtonGroup-ButtonGroup-vcMeG"><div><div class="Box-sc-g0xbh4-0" data-loading-wrapper="true"><button type="button" class="prc-Button-ButtonBase-c50BI flex-1" data-loading="false" data-no-visuals="true" data-size="medium" data-variant="default" aria-describedby=":r11:-loading-announcement"><span data-component="buttonContent" data-align="center" class="prc-Button-ButtonContent-HKbr-"><span data-component="text" class="prc-Button-Label-pTQ3x">Update branch</span></span></button></div></div><div><button data-component="IconButton" type="button" aria-label="Update branch options" aria-haspopup="true" aria-expanded="false" tabindex="0" class="prc-Button-ButtonBase-c50BI flex-0 prc-Button-IconButton-szpyj" data-loading="false" data-no-visuals="true" data-size="medium" data-variant="default" aria-describedby=":r12:-loading-announcement" id=":r12:"><svg aria-hidden="true" focusable="false" class="octicon octicon-triangle-down" viewBox="0 0 16 16" width="16" height="16" fill="currentColor" style="display: inline-block; user-select: none; vertical-align: text-bottom; overflow: visible;"><path d="m4.427 7.427 3.396 3.396a.25.25 0 0 0 .354 0l3.396-3.396A.25.25 0 0 0 11.396 7H4.604a.25.25 0 0 0-.177.427Z"></path></svg></button></div></div></div></div></div></div></section>
```

</details>

<details>
<summary>DOM with `update-pr-from-base-branch` (25.1.24)</summary>

```html
<section aria-label="Conflicts" class="border-bottom borderColor-muted" aria-describedby=":r10:"><div class="d-flex flex-column width-full overflow-hidden"><div class="MergeBoxSectionHeader-module__wrapper--f99Ts flex-column flex-sm-row flex-items-center flex-sm-items-start flex-justify-between"><div class="branch-action-btn js-immediate-updates js-needs-timeline-marker-header flex-order-2 flex-self-center"><button type="button" class="btn btn-sm rgh-update-pr-from-base-branch tooltipped tooltipped-w" aria-label="Use Refined GitHub to update the PR from the base branch">Update branch</button></div><div class="d-flex width-full"><div class="mr-2 flex-shrink-0"><div overflow="hidden" size="32" class="Box-sc-g0xbh4-0 iAmUFw"><div display="flex" size="32" class="Box-sc-g0xbh4-0 jneZjk"><svg aria-hidden="true" focusable="false" class="octicon octicon-alert-fill" viewBox="0 0 16 16" width="16" height="16" fill="currentColor" style="display: inline-block; user-select: none; vertical-align: text-bottom; overflow: visible;"><path d="M6.457 1.047c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0 1 14.082 15H1.918a1.75 1.75 0 0 1-1.543-2.575ZM8 5a.75.75 0 0 0-.75.75v2.5a.75.75 0 0 0 1.5 0v-2.5A.75.75 0 0 0 8 5Zm1 6a1 1 0 1 0-2 0 1 1 0 0 0 2 0Z"></path></svg></div></div></div><div class="d-flex flex-1 flex-column flex-sm-row gap-2"><div class="flex-1"><h3 class="Box-sc-g0xbh4-0 isSOdJ prc-Heading-Heading-6CmGO" id=":r10:">This branch is out-of-date with the base branch</h3><p class="fgColor-muted mb-0 ">Merge the latest changes from develop into this branch. This merge commit will be associated with vintagesucks.</p></div><div class="Box-sc-g0xbh4-0 prc-ButtonGroup-ButtonGroup-vcMeG"><div><div class="Box-sc-g0xbh4-0" data-loading-wrapper="true"><button type="button" class="prc-Button-ButtonBase-c50BI flex-1" data-loading="false" data-no-visuals="true" data-size="medium" data-variant="default" aria-describedby=":r11:-loading-announcement"><span data-component="buttonContent" data-align="center" class="prc-Button-ButtonContent-HKbr-"><span data-component="text" class="prc-Button-Label-pTQ3x">Update branch</span></span></button></div></div><div><button data-component="IconButton" type="button" aria-label="Update branch options" aria-haspopup="true" aria-expanded="false" tabindex="0" class="prc-Button-ButtonBase-c50BI flex-0 prc-Button-IconButton-szpyj" data-loading="false" data-no-visuals="true" data-size="medium" data-variant="default" aria-describedby=":r12:-loading-announcement" id=":r12:"><svg aria-hidden="true" focusable="false" class="octicon octicon-triangle-down" viewBox="0 0 16 16" width="16" height="16" fill="currentColor" style="display: inline-block; user-select: none; vertical-align: text-bottom; overflow: visible;"><path d="m4.427 7.427 3.396 3.396a.25.25 0 0 0 .354 0l3.396-3.396A.25.25 0 0 0 11.396 7H4.604a.25.25 0 0 0-.177.427Z"></path></svg></button></div></div></div></div></div></div></section>
```

</details>

**Comment by fregante** [comment]
The `canNativelyUpdate` selector needs to be updated too

**Comment by yutotakano** [comment]
Sorry, just want to check - is [clear-pr-merge-commit-message](https://github.com/refined-github/refined-github/blob/main/source/features/clear-pr-merge-commit-message.tsx?rgh-link-date=2024-12-05T01%3A16%3A11.000Z) supposed to clear out the merge commit for non-squash merges too, in the new PR merge box beta? (e.g. merge commit, rebase)

**Comment by fregante** [comment]
Yes

**Comment by matthewhively** [comment]
`suggest-commit-title-limit` still erroring in 25.3.19

**Comment by fregante** [comment]
I'm missing `pr-base-commit` particularly

**Comment by pazustep** [comment]
`batch-mark-files-as-viewed` is broken in the new experience, unfortunately.

**Comment by SunsetTechuila** [comment]
Everything has been fixed 🙂
